src/index.ts

```diff
@@ -188,6 +191,17 @@ export class GoogleToken {
   }
 
   private async getTokenAsync(opts: GetTokenOptions): Promise<TokenData> {
+    if (this.inFlightRequest && !opts.forceRefresh) {
+      return await this.inFlightRequest;
```

Comment:
If `this.inFlightRequest` is already a promise, can't we just return it?

Reply:
You can, but I think it's generally preferable to be explicit because it avoids a refactoring hazard. Adding and removing `await` only makes no difference here because we are not inside a `try`/`catch` block. I will remove the `await` as it's not super important.